roles/splunk/defaults/main.yml

```diff
@@ -20,7 +20,8 @@ clientName: undefined
 splunk_admin_username: admin
 splunk_admin_password: undefined # Use ansible-vault encrypt_string, e.g. ansible-vault encrypt_string --ask-vault-pass 'var_value_to_encrypt' --name 'var_name'
 splunk_configure_secret: false # If set to true, you need to update files/splunk.secret
-splunk_secret_file: splunk.secret # Used to specify your splunk.secret filename(s), files should be placed in the "files" folder of the role
+#splunk_secret_file: splunk.secret # Used to specify your splunk.secret filename(s), files should be placed in the "files" folder of the role
```

Comment:
Need to avoid having variables commented out in the default variables file.

Reply:
Please follow the existing pattern - i.e. default value of "undefined" that is checked in the task. This has the advantage of listing all valid variables in the default vars, but still not implementing them unless they've been configured via group_vars/host_vars/playbook vars.

Reply:
Fixed to match the pattern. I set both to undefined. Let me know if you want the file one returned to `splunk.secret`.